src/api/app/models/event/build.rb

```diff
@@ -54,5 +63,21 @@ def my_message_id
       mid = Digest::MD5.hexdigest("#{payload['project']}-#{payload['package']}-#{payload['repository']}-#{md5}")
       "<build-#{mid}@#{self.class.message_domain}>"
     end
+
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def maintained_project_name
+      maintenance_project = Project.get_maintenance_project
+      return unless maintenance_project
+
+      return maintenance_project.name if payload['project'].start_with?(maintenance_project.name)
+
+      # maintained projects and their sub-projects
+      return unless maintenance_project.maintained_project_names.any? { |name| payload['project'].start_with?(name) }
```

Comment:
This could be moved into `Project` also.

Reply:
true

Reply:
it is now.